.github/workflows/test-fuzzy.yml

```diff
@@ -0,0 +1,16 @@
+name: test borsh
+run-name: ${{ github.actor }}'s patch
+on: [push, pull_request]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          cache: true
+          toolchain: nightly
+      - run: |
+          cargo install cargo-fuzz
+          cargo +nightly fuzz build
+          cargo +nightly fuzz run fuzz -- -max_total_time=300
```

Comment:
This seems pretty excessive. Could we reduce to 60s? (all other tests run in 30s)